wiki/Getting_Started_Contributing_to_Wiki/en.md

```diff
@@ -0,0 +1,178 @@
+ 
+ Go to the [GitHub repository](https://github.com/ppy/osu-wiki).
+ 
+ You should see something like this.
+
+![osu-wiki Repository Snapshot](3.JPG "Snapshot of the osu!Wiki Repository")
+
+Now click the "Fork" Button on the top-right and it would prompt you on where to fork the repository.
+
+![Fork prompt dialogue](4.JPG "Fork prompt dialogue")
+
+If you have already made a fork, the Prompt will also say where was it forked.
+
+Now click at your profile and GitHub would redirect you to where the fork is.
+
+And you should see something like this.
+
+![Fork Snapshot](5.JPG "Fork Snapshot")
+
+Congratulations! You made a copy of the osu!wiki repository! You can now proceed in making changes.
+
+## Making Changes 
+
+### Through GitHub's Web Editor
+If you don't want the hassle of cloning your fork into your PC, GitHub offers Editing through your browser.
+
+Go to your desired Folder that you want to edit and pick a Markdown File to edit.
+
+_Note: Each article is grouped into folders and each folder contains markdown files arranged by language and their media.
+Please follow this format if you're creating a new Article._
+
+Now click your desired Markdown file to edit. And you should see something like this.
+
+![GitHub Preview](7.JPG "Github Preview")
+
+Now Click on the Pencil Icon to edit. And it should redirect you to the Web Editor.
+
+![GitHub's Web Editor](8.JPG "web Editor Snapshot")
+
+If you want to preview your changes, click the "Preview Changes" Tab. Click "Edit File" to go back to Editing.
+
+After making your changes, scroll down and Fill the following fields to explain your edit to other contributors.
+
+![Commit Message](9.JPG "Commit Message")
+
+You would be given a option where to save your edits.
+
+- "Commit Directly to the ``xy`` branch" will commit the changes through your branch directly.
+
+- "Create a New Branch" will automatically create a Branch for your edits. This is useful if you're making bulk edits.
+
+Pick a option and click "Commit Changes".
+
+### Through your local Machine
+
+GitHub under the hood is a Git Service. You can clone your fork in your Machine and make changes there.
+
+if you have git installed, put this in your Command Line Interface :
+```prolog
+git clone https://github.com/<username>/osu-wiki 
+```
+and this should clone your fork into your specified folder.
+
+Now after Making your changes locally, we need to commit this on your local copy.
+
+this can be done by 
+```prolog
+git add .
+git commit -m "<commit message here>"
+```
+However, in some Editors like Visual Studio Code, they provide a Git Management Interface.
+
+![Visual Studio code's Git Interface](10.JPG "Visual Studio's Git Interface")
+
+After Committing your changes, we ned to sync this on GitHub, to be able to do this, Enter the following in your Command Line Interface :
+
+```prolog
+git push origin <branch>
+```
+and this should upload your changes in GitHub.
```

Comment:
if you have your authentication details saved, it would not prompt you and use that and would skip to push